### PR TITLE
feat: [IDP-4009]: Adding statuspage icon to uicore

### PR DESCRIPTION
**Pull request by Prerna-Harness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by samriddhi-daga** [comment]
retrigger prettier

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2765

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2767

**Comment by Prerna-Harness** [comment]
retrigger build

**Comment by samriddhi-daga** [comment]
retrigger build

File: packages/icons/src/statuspage.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg width="2008" height="1461" viewBox="0 0 2008 1461" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (samriddhi-daga) [inline packages/icons/src/statuspage.svg:1]: Remove width and height and compress the svg

**Review by yashr-harness — DISMISSED** [review]

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2777

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2781

File: packages/icons/src/HarnessIcons.ts [hunk 2]
```diff
@@ -2744,6 +2745,7 @@ type HarnessIconName =
   | 'status-pending'
   | 'status-running'
   | 'status-upgrade'
+  | 'statuspage'
```
Comment (samriddhi-daga) [inline packages/icons/src/HarnessIcons.ts:2748]:
```suggestion
  | 'status-page'
```

File: packages/icons/src/HarnessIcons.ts [hunk 3]
```diff
@@ -4186,6 +4188,7 @@ const HarnessIcons: KVO<ElementType> = {
   'status-pending': StatusPending,
   'status-running': StatusRunning,
   'status-upgrade': StatusUpgrade,
+  statuspage: Statuspage,
```
Comment (samriddhi-daga) [inline packages/icons/src/HarnessIcons.ts:4191]:
```suggestion
  'status-page': Statuspage,
```

File: packages/icons/src/HarnessIcons.ts [hunk 4]
```diff
@@ -1302,6 +1302,7 @@ import StatsUpSquare from './stats-up-square.svg'
 import StatusPending from './status-pending.svg'
 import StatusRunning from './status-running.svg'
 import StatusUpgrade from './status-upgrade.svg'
+import Statuspage from './statuspage.svg'
```
Comment (samriddhi-daga) [inline packages/icons/src/HarnessIcons.ts:1305]: Please rename it to `status-page`

```suggestion
import Statuspage from './status-page.svg'
```

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2797

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2809

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2813

File: packages/icons/src/HarnessIcons.ts (hunk 4 above)
Comment (samriddhi-daga) [reply packages/icons/src/HarnessIcons.ts:1306]: Please make the changes wherever mentioned

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2821

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2825

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2831

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2833

**Review by samriddhi-daga — APPROVED** [review]
LGTM

**Review by yashr-harness — APPROVED** [review]

**Pull request merged**